templates/base/server/routes/user.py

```
@@ -0,0 +1,41 @@
+@router.get("/", response_model=List[UserResponse])
+async def get_users(user_service: UserService = Depends(get_user_service) , current_user: User = Depends(get_current_active_user)):
+    """Get all users. Returns a list of users."""
+    return await user_service.get_users(current_user)
+    
+@router.get("/me", response_model=UserResponse, status_code=status.HTTP_200_OK)
+async def get_current_user(current_user: User = Depends(get_current_active_user)):
+    """Get the current user. Returns the user associated to the current session."""
+    return current_user
+
+@router.get("/{userd_id}", response_model=UserResponse, status_code=status.HTTP_200_OK)
+async def get_user(userd_id: str, user_service: UserService = Depends(get_user_service) , current_user: User = Depends(get_current_active_user)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)